src/connectors/hits/connectHits.js

```diff
@@ -60,8 +60,14 @@ export default function connectHits(renderFn, unmountFn = noop) {
     return {
       $$type: 'ais.hits',
 
-      getConfiguration() {
-        return escapeHTML ? TAG_PLACEHOLDER : undefined;
+      getConfiguration(state) {
+        if (!escapeHTML) {
+          return state;
+        }
+
+        return state.setQueryParameters({
+          ...TAG_PLACEHOLDER,
```

Comment:
feel free to ignore this; but `setQueryParameters(TAG_PLACEHOLDER)` is an option too

Reply:
Agree with @Haroenv the spread is useless in that case.